packages/signature-controller/CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [19.1.0]
+
+### Changed
+
+- Add tracing infrastructure ([#4575](https://github.com/MetaMask/core/pull/4575))
+  - Add optional `trace` callback to constructor.
+  - Add optional `traceContext` option to `newUnsignedPersonalMessage` and `newUnsignedTypedMessage` methods.
+  - Add initial tracing of signature lifecycle.
+
```

Comment:
```suggestion
- Bump `@metamask/controller-utils` from `^11.2.0` to `^11.3.0` ([#4688](https://github.com/MetaMask/core/pull/4688))

```

Reply:
@OGPoyraz Looks like these entries got overwritten

Reply:
Updated suggestion to reflect category change.

Reply:
Done in https://github.com/MetaMask/core/pull/4688/commits/104bf0f49683ac17a730979ba6ef188ebaf729eb